packages/react/src/theme.css

```diff
@@ -1,3 +1,49 @@
+/* In-App Message theme */
+[data-knock-color-mode="light"] {
```

Comment:
Based on the Figma variables that will be available in the public in-app message designs. Vars need to be defined for each color mode to match how they're generated by Telegraph